tests/utils/test_utils_show_jwt_claims.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2024 SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import subprocess
+
+
+def test_utils_show_jwt_all_claims(astarte_env_vars):
+
+    jwt = astarte_env_vars["jwt"]
+
+    arg_list = [
+        "astartectl",
+        "utils",
+        "show-jwt-claims",
+        jwt,
+    ]
+
+    sample_data_result = subprocess.run(arg_list, capture_output=True, text=True)
+    except_output = ["a_aea", "a_rma", "a_pa", "a_ch"]
```

Comment:
Nit: I would go with a name that's a little more clear, like:
```suggestion
    expected_output = ["a_aea", "a_rma", "a_pa", "a_ch"]
```

Reply:
This suggestion applies to the other tests in this file too